TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -33,6 +33,11 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         self.reload(row: .marketPrice, animation: .none)
         self.reload(row: .estimatedCost, animation: .none)
     }
+    private var supportedOrderQuantityTypes: [OrderQuantityType] {
+        get {
+            return self.equityOrderCapabilities?.supportedOrderQuantityTypes(forAction: self.order.action, priceType: self.order.type) ?? []
```

Comment:
Let's do a rename on `equityOrderCapabilities` to whatever is in the Yahoo screens... either `instrumentOrderCapabilities` or just `orderCapabilities`. It'll just make it easier when refactoring/collapsing these tickets later.

Reply:
@falnatsheh also don't forget to update this name pls